docs/examples/cookbook/loading-spinner.md

```diff
@@ -41,7 +41,7 @@ local function Spinner(
 		BackgroundTransparency = 1,
 		Image = "rbxassetid://your-loading-spinner-image", -- replace this!
 
-		Rotation = scope:Computed(function(use)
+		Rotation = scope:Computed(function(use, scope)
 			return (use(props.CurrentTime) * SPIN_DEGREES_PER_SECOND) % 360
```

Comment:
Scope shouldn't be provided as it is never used
```suggestion
		Rotation = scope:Computed(function(use)
```

Reply:
reverted